libraries/AP_Frsky_Telem/AP_Frsky_Parameters.h

```diff
@@ -0,0 +1,49 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Param/AP_Param.h>
+
+/* Parameters defaults */
+#define FRSKY_UPLINK_ID_DEFAULT     13
```

Comment:
So this is a preference of mine and some other devs.  If these constants are only used on one place - just put them in place.  You don't need to act on this suggestion here.

Reply:
I know, but I like having defines makes code readable